src/tiled/wangcolormodel.cpp

```diff
@@ -42,9 +42,7 @@ WangColorModel::WangColorModel(TilesetDocument *tilesetDocument,
 
 QModelIndex WangColorModel::colorIndex(int color) const
 {
-    if (mWangSet)
-        Q_ASSERT(color <= mWangSet->colorCount());
-    else
+    if (!mWangSet || color > mWangSet->colorCount())
```

Comment:
Should I put the assert back now that the script error is implemented? Or do we just want to make absolutely sure no one can trigger the assert by scripting

Reply:
I think it's fine to leave it like this now.